apps/web/lib/mutations/bookings/create-recurring-booking.ts

```diff
@@ -0,0 +1,21 @@
+import * as fetch from "@lib/core/http/fetch-wrapper";
+import { BookingCreateBody, BookingResponse } from "@lib/types/booking";
+
+type ExtendedBookingCreateBody = BookingCreateBody & { noEmail?: boolean };
+
+const createRecurringBooking = async (data: BookingCreateBody[]) => {
+  return Promise.all(
+    data.map((booking, key) => {
+      if (key === 0) {
+        return fetch.post<ExtendedBookingCreateBody, BookingResponse>("/api/book/event", booking);
+      } else {
+        return fetch.post<ExtendedBookingCreateBody, BookingResponse>("/api/book/event", {
+          ...booking,
+          noEmail: true,
```

Comment:
Added a flag to avoid sending emails for every booking, just for the first

Reply:
This flag can be leveraged to add a configuration to skip receiving emails cc: @agustif